airflow-core/src/airflow/ui/src/router.tsx

```diff
@@ -60,13 +60,13 @@ import { XCom } from "src/pages/XCom";
 
 import { client } from "./queryClient";
 
-const iframeRoute = {
+const pluginRoute = {
   // The following iframe sandbox setting is intentionally less restrictive.
   // This is considered safe because the framed content originates from the Plugins,
   // which is part of the deployment of Airflow and trusted as per our security policy.
   // https://airflow.apache.org/docs/apache-airflow/stable/security/security_model.html
   // They are not user provided plugins.
```

Comment:
This comment is not relevant anymore and should be removed or moved to the appropriate place if we come back to a allow-same-origin + allow-script sandbox somewhere.